### PR TITLE
Use separate folders for agents logs directory

### PR DESCRIPTION
**Pull request by matrixik**
Previously used /var/log/monasca/ was giving problems because metric
agent have custom `mon-agent` user and log agent need to run
as `root` user.
Now we are using two separate `/var/log/monasca-agent/` and
`/var/log/monasca-log-agent` directories.

### CODE REVIEW

**Comment by witekest** [comment]
Log file locations are also specified in `agent.yaml`

**Comment by matrixik** [comment]
But this file is created by `monasca-setup` and we simply relay all configuration to it from user.

**Comment by matrixik** [comment]
Here: https://github.com/monasca/monasca-agents-installer/blob/master/configure_metrics_agent.sh#L74

**Comment by witekest** [comment]
So where do you control the path for log locations configured in `agent.yaml`? Default value is [/var/log/monasca/agent](https://github.com/openstack/monasca-agent/blob/master/monasca_setup/main.py#L263). Or am I missing something?

**Comment by matrixik** [comment]
OK, fixed, now `monasca-setup` always have overwritten `--log_dir` with proper folder.

File: configure_metrics_agent.sh [hunk 1]
```diff
@@ -98,7 +100,7 @@ supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 minfds = 1024
 minprocs = 200
 loglevel = info
-logfile = /var/log/monasca/agent/supervisord.log
+logfile = ${MON_CUSTOM_AGENT_LOG_DIR}/supervisord.log
```
Comment (witekest) [inline configure_metrics_agent.sh]: I guess it should be ${MON_AGENT_LOG_DIR}

Comment (matrixik) [reply configure_metrics_agent.sh]: Ugh, thx for spotting.
Done.

**Review by witekest — APPROVED** [review]

**Review by adriancz — APPROVED** [review]

**Pull request merged**